src/components/checkout/CheckoutProductModal.tsx

```diff
@@ -53,113 +55,117 @@ export type CheckoutProductModalProps = {
   }) => React.ReactElement
 }
 const CheckoutProductModal: React.VFC<CheckoutProductModalProps> = ({
+  defaultProductId,
   renderTrigger,
   renderProductSelector,
-  paymentType,
-  defaultProductId,
-  isProductPhysical,
   warningText,
   startedAt,
-  member,
   shippingMethods,
 }) => {
   const { formatMessage } = useIntl()
-  const history = useHistory()
   const [sharingCode] = useQueryParam('sharing', StringParam)
-  const { enabledModules, settings } = useApp()
-  const updateMemberMetadata = useUpdateMemberMetadata()
+  const history = useHistory()
   const { isOpen, onOpen, onClose } = useDisclosure()
-  const [groupBuying, setGroupBuying] = useState<{
-    memberIds: string[]
-    withError: boolean
-  }>({ memberIds: [], withError: false })
+  const { enabledModules, settings } = useApp()
+  const { currentMemberId } = useAuth()
+  const { member: currentMember } = useMember(currentMemberId || '')
 
-  // payment information
-  const cachedPaymentInfor: {
-    shipping: ShippingProps
-    invoice: InvoiceProps
-    payment: PaymentProps
+  // checkout
+  const [productId, setProductId] = useState(defaultProductId)
+  const { target } = useSimpleProduct({ id: productId, startedAt })
+
+  // cart information
+  const memberCartInfor: {
+    shipping?: ShippingProps | null
+    invoice?: InvoiceProps | null
```

Comment:
NITS: `memberCartInfor` ---> `memberCartInfo`

Reply:
the two abbreviations are both fine, however, we use `info` more